src/coin/skycoin/models/account.go

```
@@ -66,13 +66,26 @@ func (wlt RemoteWallet) ScanUnspentOutputs() core.TransactionOutputIterator { //
 	return nil
 }
 
-func (wlt RemoteWallet) ListTransactions() core.TransactionIterator { //------TODO
+func (wlt RemoteWallet) ListTransactions() core.TransactionIterator { 
 	return nil
 }
 func (wlt RemoteWallet) ListPendingTransactions() core.TransactionIterator { //------TODO
 	return nil
 }
 
+func (wlt RemoteWallet) ListPendingTransactions() core.TransactionIterator { 
```

This function should return an error on failure